### PR TITLE
Bug 1207566 - [wc] gaia_switch

### PR DESCRIPTION
**Pull request by gabtoubl**
https://bugzilla.mozilla.org/show_bug.cgi?id=1207566

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=476c459d395839bc853830e5c19a9a8e6c767a58

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=07cbd583f107c64c34c9d756851550d9d7ddf124

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=40efa8e2c0d8e394ad23b4e5289db9bf05925e94

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d823a40002d59190100679f9995a0bfadc12bee5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=141eaffbec1c188088bfc15ff1b09d6c7a81e107

File: build/csslint/xfail.list [hunk 1]
```diff
@@ -87,10 +88,13 @@ shared/style/confirm.css 7 0
 shared/style/contacts/contacts_buttons.css 0 7
 shared/style/date_selector.css 1 0
 shared/style/dialer/dialer.css 0 1
+shared/style/edit_mode.css 0 2
 shared/style/edit_mode.css 1 0
```
Comment (fabi1cazenave) [inline build/csslint/xfail.list]: conflict for edit_mode.css — I guess the line you added is useless?

File: build/csslint/xfail.list [hunk 2]
```diff
@@ -87,10 +88,13 @@ shared/style/confirm.css 7 0
...
+shared/style/headers.css 0 0
```
Comment (fabi1cazenave) [inline build/csslint/xfail.list]: let’s leave headers.css alone, it has zero warning and zero error anyway

File: build/csslint/xfail.list [hunk 3]
```diff
@@ -87,10 +88,13 @@ shared/style/confirm.css 7 0
...
 shared/style/input_areas.css 1 0
 shared/style/lists.css 5 0
 shared/style/object_menu.css 2 0
 shared/style/progress_activity.css 1 0
 shared/style/time_selector.css 1 0
+shared/style/value_selector.css 0 1
 shared/style/value_selector.css 3 1
```
Comment (fabi1cazenave) [inline build/csslint/xfail.list]: another conflict for value_selector, I guess the latter prevail

File: build/csslint/xfail.list (hunk 1 above)
Comment (gabtoubl) [inline build/csslint/xfail.list]: It was already there in the document, I didn't add it, I just sort the file so it ended here.

Still, as you said the latter rule override the first, so it isn't needed anymore

File: build/csslint/xfail.list (hunk 3 above)
Comment (gabtoubl) [inline build/csslint/xfail.list]: You're right, tested, incoming in the next push

File: build/csslint/xfail.list (hunk 2 above)
Comment (gabtoubl) [inline build/csslint/xfail.list]: Again, not me who added it, but it seems unnecessary indeed.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5b38646d2cd4a8475c78d3c881abeef08fc55442

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=eb1e06a832ba8abcf7839bf390baed9d727c8018

**Pull request merged**